src/main.test.js

```diff
@@ -92,30 +92,30 @@ test("Untyped environmental variable", () => {
 
 test("Multiple invalid variables and types", () => {
   const testEnv3 = "DATABASE_URLαα: αα\nADMIN_EMAIL:\n1DEBUG";
-  expect(envSpecToHTML(testEnv3)).toEqual("Error:Wrong Syntax");
+  expect(envSpec.parse(testEnv3)).rejects.toEqual("Error:Wrong Syntax");
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#rejects)).